src/functions/sendCandidateResults/application/service/__tests__/template-id-provider.spec.ts

```diff
@@ -68,4 +158,69 @@ describe('get-template-id-provider', () => {
     });
   });
 
+  describe('isVocationalCategory', () => {
+    it('should return true if category is in vocational category array', () => {
+      expect(isVocationalCategory(TestCategory.C)).toBe(true);
+      expect(isVocationalCategory(TestCategory.C1)).toBe(true);
+      expect(isVocationalCategory(TestCategory.CE)).toBe(true);
+      expect(isVocationalCategory(TestCategory.C1E)).toBe(true);
+    });
+
+    it('should return false when category is not in the vocational category array', () => {
+      expect(isVocationalCategory(TestCategory.B)).toBe(false);
+      expect(isVocationalCategory(TestCategory.BE)).toBe(false);
+    });
+  });
+
+  describe('getTemplateString', () => {
+    it('should return true if category is in vocational category array', () => {
+      const conductedLanguage: ConductedLanguage = 'English';
+      const communicationMethod: CommunicationMethod = 'Email';
+      const activityCode: ActivityCode = '1';
+      expect(getTemplateString(conductedLanguage, communicationMethod, activityCode))
+        .toBe('englishEmailPassTemplateId');
+    });
+    it('should return true if category is in vocational category array', () => {
+      const conductedLanguage: ConductedLanguage = 'Cymraeg';
+      const communicationMethod: CommunicationMethod = 'Email';
+      const activityCode: ActivityCode = '1';
+      expect(getTemplateString(conductedLanguage, communicationMethod, activityCode))
+        .toBe('welshEmailPassTemplateId');
+    });
+    it('should return true if category is in vocational category array', () => {
+      const conductedLanguage: ConductedLanguage = 'English';
+      const communicationMethod: CommunicationMethod = 'Post';
+      const activityCode: ActivityCode = '1';
+      expect(getTemplateString(conductedLanguage, communicationMethod, activityCode))
+        .toBe('englishLetterPassTemplateId');
+    });
+    it('should return true if category is in vocational category array', () => {
```

Comment:
from lines 175 - end of file - large number of it calls with 'should return true if category is in vocational category array' this description does not match the tests.

Reply:
yeah just realised ive duplicated the descriptions, ill update now!